### PR TITLE
Funding - adds base file for funding proposal

### PR DESCRIPTION
**Pull request by bordumb**
## Overview
This is a PR to create the final Governance Proposal - [as outlined here](https://radicle.community/t/readme-radicle-governance-process/526) - for the Grants Program

It should follow the CLI tooling here:
https://github.com/radicle-dev/radgov

## Changes 
* Notes covering the logic in plain English
* Script to send 1,000,000 USDC from Radicle Treasury to the Radicle Grants Program multi-sig
* Script to send 44,108 RAD from Radicle Treasury to the Radicle Grants Program multi-sig

### CODE REVIEW

File: governance/funding/start_funding.proposal [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+Please see discussions here:
+* Temperature Check: https://radicle.community/t/temperature-check-radicle-grants-program/2461/8
+* Formal Discussion (V1): https://radicle.community/t/formal-review-radicle-grants-program-v1/2548
+* Formal Discussion (V2): https://radicle.community/t/formal-review-radicle-grants-program-v2/2582/22
+* Snapshot Pull (94% Approval): https://snapshot.org/#/gov.radicle.eth/proposal/0xe9158c0e656918f207f76af0b05feefce2cf07c02e582944825373eebd4ddaff
+
+## Notes
+* `0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48` is the USDC contract (https://etherscan.io/token/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48#tokenTrade).
+* `0x31c8EAcBFFdD875c74b94b077895Bd78CF1E64A3` is the RAD contract (https://etherscan.io/token/0x31c8EAcBFFdD875c74b94b077895Bd78CF1E64A3#readContract).
+* `0x8da8f82d2bbdd896822de723f55d6edf416130ba` is the Radicle Foundation address.
```
Comment (abbey-titcomb) [inline governance/funding/start_funding.proposal]:
```suggestion
* `0x8da8f82d2bbdd896822de723f55d6edf416130ba` is the Radicle Treasury address.
```

File: governance/funding/start_funding.proposal [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+1. Transfer 1.0M USDC from the Radicle Foundation to the Radicle Grant Program's multi-sig
+2. Transfer 44,108 RAD from the Radicle Foundation to the Radicle Grant Program's multi-sig
+
+**How was this proposal discussed?**
+Please see discussions here:
+* Temperature Check: https://radicle.community/t/temperature-check-radicle-grants-program/2461/8
+* Formal Discussion (V1): https://radicle.community/t/formal-review-radicle-grants-program-v1/2548
+* Formal Discussion (V2): https://radicle.community/t/formal-review-radicle-grants-program-v2/2582/22
+* Snapshot Pull (94% Approval): https://snapshot.org/#/gov.radicle.eth/proposal/0xe9158c0e656918f207f76af0b05feefce2cf07c02e582944825373eebd4ddaff
```
Comment (abbey-titcomb) [inline governance/funding/start_funding.proposal:13]:
```suggestion
* Snapshot Poll (94% Approval): https://snapshot.org/#/gov.radicle.eth/proposal/0xe9158c0e656918f207f76af0b05feefce2cf07c02e582944825373eebd4ddaff
```

File: governance/funding/start_funding.proposal [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# Radicle Grants Wave 1 - Start Funding
+
+If executed, this proposal will:
+
+1. Transfer 1.0M USDC from the Radicle Foundation to the Radicle Grant Program's multi-sig
```
Comment (abbey-titcomb) [inline governance/funding/start_funding.proposal:5]:
```suggestion
1. Transfer 1.0M USDC from the Radicle Treasury to the Radicle Grant Program's multi-sig
```

File: governance/funding/start_funding.proposal [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+2. Transfer 44,108 RAD from the Radicle Foundation to the Radicle Grant Program's multi-sig
```
Comment (abbey-titcomb) [inline governance/funding/start_funding.proposal:6]:
```suggestion
2. Transfer 44,108 RAD from the Radicle Treasury to the Radicle Grant Program's multi-sig
```

**Review by abbey-titcomb — CHANGES_REQUESTED** [review]
@bordumb - let's just make sure the wording is all correct here before deploying.

File: governance/funding/start_funding.proposal (hunk 3 above)
Comment (bordumb) [reply governance/funding/start_funding.proposal:5]: Done

File: governance/funding/start_funding.proposal (hunk 2 above)
Comment (bordumb) [reply governance/funding/start_funding.proposal:13]: Done

File: governance/funding/start_funding.proposal (hunk 4 above)
Comment (bordumb) [reply governance/funding/start_funding.proposal:6]: Done

**Review by abbey-titcomb — APPROVED** [review]
